tasks/task1437_doqa_cooking_question_generation.json

```diff
@@ -0,0 +1,5525 @@
+{
+	"Contributors": [
+		"Kuntal Kumar Pal"
+	],
+	"Source": [
+		"DoQA: (dataset: http://www.ixa.eus/node/12931, paper: https://arxiv.org/pdf/2005.01328.pdf)"
+	],
+	"Input_language": [
+		"English"
+	],
+	"Output_language": [
+		"English"
+	],
+	"Instruction_language": [
+		"English"
+	],
+	"Categories": [
```

Comment:
Please update the categories and add the domain field based on the task-hierarchy.